scripts/configuration/testProjects.csv

```diff
@@ -1,4 +1,5 @@
 EndToEnd
+dotnet.Tests
```

Comment:
I'm completely open to any name.  @piotrpMSFT suggested "dotnet.Tests" because these are actually testing the dotnet command itself, and not a sub-command.